Dockerfile

```diff
@@ -52,41 +46,18 @@ RUN addgroup --system --gid ${CISA_GID} ${CISA_GROUP} \
 ###
 ENV DEPS \
     ca-certificates \
+    libc6-compat \
     openssl \
-    py-pip
-RUN apk --no-cache --quiet add ${DEPS}
-
-###
-# Make sure pip and setuptools are the latest versions
-#
-# Note that we use pip --no-cache-dir to avoid writing to a local
-# cache.  This results in a smaller final image, at the cost of
-# slightly longer install times.
-###
-RUN pip install --no-cache-dir --upgrade pip setuptools
+    wget
 
-WORKDIR ${CISA_HOME}
+RUN apk --no-cache --quiet add ${DEPS}
 
-###
-# Install Python dependencies
-#
-# Note that we use pip --no-cache-dir to avoid writing to a local
-# cache.  This results in a smaller final image, at the cost of
-# slightly longer install times.
-###
-RUN wget --output-document sourcecode.tgz \
-    https://github.com/cisagov/skeleton-python-library/archive/v${VERSION}.tar.gz \
-    && tar --extract --gzip --file sourcecode.tgz --strip-components=1 \
-    && pip install --no-cache-dir --requirement requirements.txt \
-    && ln -snf /run/secrets/quote.txt src/example/data/secret.txt \
-    && rm sourcecode.tgz
+# Install Con-PCA-Tasks binary file
+RUN wget https://github.com/cisagov/con-pca-tasks/releases/download/v0.0.1/pca-linux-amd64
+RUN mv pca-linux-amd64 /bin/pca
```

Comment:
Why are you only using the amd64 release here?  What about the other platforms?

Reply:
@dav3r we currently use a `db.r5.xlarge` instance which is running on x86-64 architecture to run our other containers in Con-PCA. We're probably going to spin up a similar instance for this project as well.

Also I'm assuming its more common that users pulling this image will build a container on an x86-64 machine. And those who do have arm64 will be able to run it anyways due to emulation. 

I don't think we need other platforms for this image build but if one would like to run an executable directly, i have other platforms supported with each release here: https://github.com/cisagov/con-pca-tasks/releases/tag/v0.0.1

Please let me know your thoughts or how you'd like to me to go forward. Thanks!

Reply:
@dav3r resolved with [8221b09](https://github.com/cisagov/con-pca-tasks-docker/pull/2/commits/8221b092804fd499d96dd73d404f8ebe49851494)

I implemented Nick's solution to choose the correct build based on the target's arch type ^

Reply:
Looks like this change was done in https://github.com/cisagov/con-pca-tasks-docker/pull/2/commits/efa61943bef119a83359dbf421dd50fc846e52a2, not https://github.com/cisagov/con-pca-tasks-docker/pull/2/commits/8221b092804fd499d96dd73d404f8ebe49851494.